notebooks/en/chain_of_verification.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #  &lt;&gt;: Update verification question that takes in entity as a parameter
</pre><p>Another TODO - is it needed here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/29/discussion/#comment-1494821417'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>